hazelcast/src/main/java/com/hazelcast/query/impl/predicates/NotEqualPredicate.java

```diff
@@ -52,12 +45,7 @@ public boolean isIndexed(QueryContext queryContext) {
 
     @Override
     public Set<QueryableEntry> filter(QueryContext queryContext) {
-        Index index = getIndex(queryContext);
```

Comment:
I removed this code as the `NotEqualsPredicates::isIndexed` always returns false. It's not strictly related to this changeset, I followed the boy-scout rule.